"old" field in "delete" event changed since v5.4.0 (#7898)
pingcap/docs: ticdc/ticdc-canal-json.md

    ],
    "old": null,
}
```

## Changes in TiCDC Canal-JSON

> ### Changes in the `Old` field of the `Delete` event message

Reviewer: ```suggestion
### Changes in the `Old` field of the `Delete` events
```

> From v5.4.0, the `old` field of the `Delete` event message has changed.

Reviewer: ```suggestion
From v5.4.0, the `old` field of the `Delete` events has changed.
```

> The following is an example of the `Delete` event message. Before v5.4.0, the `old` field contains the same content as the "data" field. In v5.4.0 and later versions, the `old` field is set to null. You can get the deleted data by using the "data" field.

Reviewer: ```suggestion
The following is a `Delete` event message. Before v5.4.0, the `old` field contains the same content as the "data" field. In v5.4.0 and later versions, the `old` field is set to null. You can get the deleted data by using the "data" field.
```
